vispy/app/tests/test_app.py

```diff
@@ -226,19 +226,19 @@ def fake(event):
         frag = "void main (void) {gl_FragColor = pos;}"
         program = Program(vert, frag)
         assert_raises(RuntimeError, program.glir.flush, context.shared.parser)
-        
```

Comment:
FYI these whitespace changes have been a hassle for me when keeping my open PR up to date.